e2e/cypress/integration/system_console/revoke_all_sessions.js

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+const buttonTheme = {backgroundColor: 'rgba(0, 0, 0, 0.7)', color: 'rgb(255, 255, 255)'};
+
+describe('System Console', () => {
+    it('SC17020 - Revoke All Sessions from System Console', () => {
+        // # Login as System Admin
+        cy.apiLogin('sysadmin').as('sysadmin');
+
+        cy.visit('/admin_console/user_management/users');
+
+        // * Verify the presence of Revoke All Sessions button
+        cy.get('#revoke-all-users').should('be.visible').and('have.css', 'background-color', buttonTheme.backgroundColor).
+            and('have.css', 'color', buttonTheme.color);
+
+        cy.get('#revoke-all-users').click();
+
+        // * Verify the confirmation message when users clicks on the Revoke All Sessions button
+        cy.get('.modal-title').should('be.visible').should('contain', 'Revoke all sessions in the system');
```

Comment:
`#confirmModalLabel` would be the preferred selector here.

Reply:
Also, for directly comparing text use `have.text` vs `contain`. Line 27 does this as well.

Reply:
Updated the test accordingly.